DQM/CTPPS/test/diamond_dqm_test_cfg.py

```diff
@@ -30,47 +28,31 @@
 
 # raw data source
 process.source = cms.Source("PoolSource",
-    # replace '*.root',',' with the source file you want to use
     fileNames = cms.untracked.vstring(
-    *(
-'/store/data/Run2017C/ZeroBias/AOD/PromptReco-v2/000/300/088/00000/469D8C89-1477-E711-A6A4-02163E01190C.root',
-     )
+'/store/data/Run2018D/ZeroBias/AOD/12Nov2019_UL2018_rsb-v1/280000/FE61A0D8-CEDC-2142-81AA-06301F452792.root',
     ),
 )
 
-
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_hlt_relval', '')
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_hlt_relval', '')
+process.GlobalTag = GlobalTag(process.GlobalTag, '120X_dataRun2_v2', '')
 
 # raw-to-digi conversion
 process.load("EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff")
 
 # local RP reconstruction chain with standard settings
 process.load("RecoPPS.Configuration.recoCTPPS_cff")
-
-# rechits production
-process.load('RecoPPS.Local.ctppsDiamondRecHits_cfi')
-
-# local tracks fitter
-process.load('RecoPPS.Local.ctppsDiamondLocalTracks_cfi')
-
-# pixel
-process.load('RecoPPS.Local.ctppsPixelLocalTracks_cfi')
-
+process.load('Geometry.VeryForwardGeometry.geometryRPFromDD_2021_cfi')
```

Comment:
Why are you using the Run 3 geometry read from the XML file? Was there a specific reason for this choice?

Reply:
> Why are you using the Run 3 geometry read from the XML file? Was there a specific reason for this choice?

Yes, with run3 geometry dqm should produce empty plots for the second station so this is included for validation purposes.

Reply:
@malbouis 
As you know the TAG for a partial Run3 geometry has been released only recently